example/cmd/device-simple/res/configuration.toml

```diff
@@ -81,7 +81,7 @@ RetryWaitPeriod = "1s"
   [[DeviceList.AutoEvents]]
     Frequency = '10s'
     OnChange = false
-    SourceName = 'Switch'
+    SourceName = 'SwitchButton'
```

Comment:
You don't need to make this change if it is RW

Reply:
Modified.